projects/views.py

```diff
@@ -0,0 +1,306 @@
+from django.views.generic import ListView, DetailView, CreateView, UpdateView, DeleteView
+from .models import *
+from django.http import HttpResponseRedirect
+from datetime import datetime
+import reversion
+
+
+# Create your views here.
+class IndexView(ListView):
+    template_name = 'projects/index.html'
+    context_object_name = 'projects'
+
+    def get_queryset(self):
+        return ProjectDetail.objects.exclude(status='ARCHIVED')
+
+
+# DETAILS
+class ProjectDetailView(DetailView):
+    model = ProjectDetail
+    context_object_name = 'pd'
```

Comment:
What's wrong with just `project` rather than `pd`? Try and be descriptive in variable names.